cmd/create/cluster/cmd.go

```diff
@@ -4134,3 +4134,13 @@ func getMachinePoolRootDisk(r *rosa.Runtime, cmd *cobra.Command, version string,
 func clusterHasLongNameWithoutDomainPrefix(clusterName, domainPrefix string) bool {
 	return domainPrefix == "" && len(clusterName) > ocm.MaxClusterDomainPrefixLength
 }
+
+func delete_empty(s []string) []string {
```

Comment:
```suggestion
func filterEmpty(s []string) []string {
```

Besides the name, I think it’s kind of weird to have a general helper function in here. Is there not a better place to put this?